lib/src/main/java/com/stepango/rxdatabindings/MaybeExt.kt

```diff
@@ -315,4 +315,4 @@ inline fun <T : Any, R : Any> Maybe<T>.safeSetTo(
 inline fun <T : Any, R : Parcelable> Maybe<T>.safeSetTo(
         field: ObservableParcelable<in R>,
         crossinline transformer: (T) -> R?
-): Maybe<T> = doOnSuccess { tValue -> transformer(tValue)?.let { field.set(it) } }
+): Maybe<T> = doOnSuccess { tValue -> transformer(tValue)?.let(field::set) }
```

Comment:
Should we add an option with the default value if transformation is null?

Reply:
I think as the method is `safeSetTo` it is understandable that only non-null values will be set. And if the user wants a default, they can implement it in their `transformer`.